src/main/java/seedu/address/logic/parser/RegisterCommandParser.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_EMAIL;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+
+import java.util.stream.Stream;
+
+import seedu.address.logic.commands.RegisterCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.borrower.Borrower;
+import seedu.address.model.borrower.Email;
+import seedu.address.model.borrower.Name;
+import seedu.address.model.borrower.Phone;
+
+/**
+ * Parses input arguments and creates a new RegisterCommand object
+ */
+public class RegisterCommandParser implements Parser<RegisterCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the RegisterCommand
+     * and returns an RegisterCommand object for execution.
+     *
+     * @return RegisterCommand object.
+     * @throws seedu.address.logic.parser.exceptions.ParseException if the user input does not conform the
```

Comment:
Should this just be @throws ParseException

Reply:
Yes it should!